addons/core/initSettings.sqf

```diff
@@ -0,0 +1,80 @@
+// ADVANCED SETTINGS
```

Comment:
``#include "script_macros.hpp"`` here? Currently this won't compile with ArmaScriptCompiler

Reply:
Should be resolved.